### PR TITLE
Fix branch name in ci

### PR DESCRIPTION
**Pull request by braingram**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 - Remove duplicate unit manifests [#5]
 - Increase setuptools version [#6]
+- Fix branch name in ci [#7]
```
Comment (WilliamJamieson) [inline CHANGES.rst]:
```suggestion
```

CI changes do not need changelogs

**Review by WilliamJamieson — CHANGES_REQUESTED** [review]

File: CHANGES.rst (hunk 1 above)
Comment (WilliamJamieson) [reply CHANGES.rst]: This is what the `no-changelog-entry-needed` label is for.

Comment (braingram) [reply CHANGES.rst]: What is the threshold for a change that should be added to the changelog?

Comment (WilliamJamieson) [reply CHANGES.rst]: It effects user code.

**Review by WilliamJamieson — APPROVED** [review]
Please squash out the changelog commits, before I merge this.

**Pull request closed without merging**